src/Data/API/Tools/DataTypeable.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Data.API.Tools.DataTypeable (
```

Comment:
I don't think we need to add this as a separate tool. How about we instead make it easier to configure the default derived instances, by moving `defaultDerivedClasses` to `ToolSettings`?

Reply:
@adamgundry sounds like a good idea to me, I'll explore that.